embedded-graphics/src/fonts/font24x32.rs

```diff
@@ -0,0 +1,285 @@
+///
+/// ```rust
+/// use embedded_graphics::prelude::*;
+/// use embedded_graphics::fonts::Font24x32;
+/// use embedded_graphics::text_24x32;
+/// # use embedded_graphics::mock_display::MockDisplay;
+/// # use embedded_graphics::pixelcolor::BinaryColor;
+/// # let mut display: MockDisplay<BinaryColor> = MockDisplay::default();
+///
+/// // Use struct methods directly
+/// display.draw(Font24x32::render_str("Hello Rust!"));
+///
+/// // Use a macro instead
+/// display.draw(text_24x32!("Hello Rust!"));
+/// ```
+///
+/// ## Translate text by (20px, 30px)
+///
+/// ```rust
+/// use embedded_graphics::prelude::*;
+/// use embedded_graphics::fonts::Font24x32;
+/// # use embedded_graphics::mock_display::MockDisplay;
+/// # use embedded_graphics::pixelcolor::BinaryColor;
+/// # let mut display: MockDisplay<BinaryColor> = MockDisplay::default();
+///
+/// display.draw(
+///     Font24x32::render_str("Hello Rust!").translate(Point::new(20, 30))
+/// );
+/// ```
+///
+/// ## Add some styling to the text
+///
+/// Use [any method provided by the `WithStyle` trait](../style/trait.WithStyle.html#required-methods).
+/// Properties like `fill` or `stroke` passed to the `text_24x32` macro are converted into method
+/// calls verbatim.
+///
+/// ```rust
+/// use embedded_graphics::prelude::*;
+/// use embedded_graphics::text_24x32;
+/// use embedded_graphics::fonts::Font24x32;
+/// use embedded_graphics::pixelcolor::Rgb565;
+/// # use embedded_graphics::mock_display::MockDisplay;
+/// # let mut display = MockDisplay::default();
+///
+/// display.draw(text_24x32!(
+///     "Hello Rust!",
+///     fill = Some(Rgb565::BLUE),
+///     stroke = Some(Rgb565::YELLOW)
+/// ));
+///
+/// display.draw(
+///     Font24x32::render_str("Hello Rust!")
+///         .translate(Point::new(20, 30))
+///         .fill(Some(Rgb565::BLUE))
+///         .stroke(Some(Rgb565::YELLOW)),
+/// );
+/// ```
+///
+/// [`text_24x32`]: ../macro.text_24x32.html
+pub type Font24x32<'a, C> = FontBuilder<'a, C, Font24x32Conf>;
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::fonts::Font;
+    use crate::geometry::{Dimensions, Point, Size};
+    use crate::mock_display::MockDisplay;
+    use crate::pixelcolor::BinaryColor;
+    use crate::style::WithStyle;
+    use crate::transform::Transform;
+    use crate::Drawing;
+
+    #[test]
+    fn off_screen_text_does_not_infinite_loop() {
+        let text: Font24x32<BinaryColor> = Font24x32::render_str("Hello World!")
+            .stroke(Some(BinaryColor::On))
+            .fill(Some(BinaryColor::Off))
+            .translate(Point::new(5, -32));
+
+        assert_eq!(text.into_iter().count(), 24 * 32 * "Hello World!".len());
+    }
+
+    #[test]
+    fn text_dimensions() {
+        let hello: Font24x32<BinaryColor> = Font24x32::render_str("Hello World!");
+        let empty: Font24x32<BinaryColor> = Font24x32::render_str("");
+
+        assert_eq!(hello.size(), Size::new(288, 32));
```

Comment:
nit: 288 seems to be pretty arbitrary. Recommend creating a var out of `"Hello World!"` so that this number can be derived from the str length (Same for `text_corners` test)

Reply:
FWIW there's an example [here](https://github.com/jamwaffles/embedded-graphics/blob/master/embedded-graphics/src/fonts/font6x8.rs#L111).

Reply:
Should I fix all of the fonts then? All of the other fonts have tests with magic numbers in the same way mine does:
```rust
    #[test]
    fn text_dimensions() {
        let hello: Font6x8<BinaryColor> = Font6x8::render_str("Hello World!");
        let empty: Font6x8<BinaryColor> = Font6x8::render_str("");

        assert_eq!(hello.size(), Size::new(72, 8));
        assert_eq!(empty.size(), Size::new(0, 0));
    }
```

Reply:
I went ahead and refactored all of the font tests to not have significant magic numbers. Let me know what you think!